config-templates/redhat/odf-local/4.7/custom-parameters.json

```diff
@@ -1,50 +1,42 @@
 [
   {
-    "description": "Specify a name for your ODF storage cluster.",
-    "displayname": "Cluster Name",
+    "description": "A unique name for your custom resource. Note that Kubernetes resources can't contain capital letters or special characters. Enter a name that uses only lowercase letters, numbers, hyphens or periods.",
+    "displayname": "OcsCluster name",
     "name": "ocs-cluster-name",
     "required": "true"
   },
   {
-    "description": "The local device path of the OSD.",
-    "displayname": "Osd Device Path",
+    "description": "The local paths on your worker nodes to the volumes you want to use for the OSD pods.",
+    "displayname": "Local paths to OSD volumes.",
     "name": "osd-device-path",
     "place-holder": "/dev/sdc,/dev/sde",
     "required": "true",
     "type": "csv"
   },
   {
-    "description": "The local device path of the MON.",
-    "displayname": "Mon device Path",
-    "name": "mon-device-path",
-    "place-holder": "/dev/sdc,/dev/sde",
-    "required": "true",
-    "type": "csv"
-  },
-  {
-    "description": "The number of OSDs. The default value is '1'.",
-    "displayname": "Num of osd",
+    "description": "The number OSD volumes that you want to provision. The total storage available to your apps is equal to the volume size (osd-size) multiplied by the number of volumes (num-of-osd). The default value is '1'.",
+    "displayname": "Number of OSD volumes.",
```

Comment:
If we make this `number of OSD volumes`, will they input 3 since they've added 3 volumes?